plugins/woocommerce-admin/client/inbox-panel/test/index.js

```diff
@@ -22,43 +25,75 @@ jest.mock( '@wordpress/data', () => {
 	};
 } );
 
-jest.mock( '@woocommerce/tracks', () => ( { recordEvent: jest.fn() } ) );
+jest.mock( '@woocommerce/experimental', () => {
+	// Require the original module to not be mocked...
+	const originalModule = jest.requireActual( '@woocommerce/experimental' );
+
+	return {
+		__esModule: true, // Use it when dealing with esModules
+		...originalModule,
+		InboxNoteCard: jest.fn().mockImplementation( ( { note } ) => {
+			return <div>{ note.id }</div>;
+		} ),
+	};
+} );
 
+// be wary that the notes are modified inside various tests
```

Comment:
We should update the tests to not do this! From a quick glance, it looks like you could modify the tests to first make a deep copy of the `notes` array (copying the items in the array as well) and operate with the copy.